src/ecdh.rs

```diff
@@ -16,12 +16,16 @@
 //! Support for shared secret computations
 //!
 
-use core::ptr;
+use core::{fmt, ptr};
 use core::ops::{FnMut, Deref};
 
 use key::{SecretKey, PublicKey};
 use ffi::{self, CPtr};
 use secp256k1_sys::types::{c_int, c_uchar, c_void};
+use to_hex;
+
+/// The buffer size (in bytes) used by `SharedSecret`.
+const BUF_SIZE: usize = 256;
```

Comment:
Unrelated but I have hard time understanding why shared secret would be larger than 32 bytes (256 bits). Any hints?

Reply:
I've done #402 to explore this.